components/camel-pgevent/pom.xml

```diff
@@ -64,6 +65,94 @@
             <artifactId>log4j-slf4j-impl</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-testcontainers-junit5</artifactId>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
 
+    <profiles>
+        <profile>
+            <id>testcontainers-skip-tests</id>
```

Comment:
Maybe it's better to have the component name too here

Reply:
OK

Reply:
done !!